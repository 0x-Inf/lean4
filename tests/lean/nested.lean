(**
cmd_macro("Simple",
          { macro_arg.String },
          function (env, str)
             print("OUTPUT: " .. str)
          end
)

parse_lean_cmds([[
   Simple "foo"
]])
**)

Simple "testing"

(**
parse_lean_cmds([[
   Simple "bla"
]])
**)
