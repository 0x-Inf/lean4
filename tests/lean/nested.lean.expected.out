  Set: pp::colors
  Set: pp::unicode
OUTPUT: foo
OUTPUT: testing
OUTPUT: bla
